finance-tracker/app/views/devise/registrations/edit.html.erb

```diff
@@ -1,37 +1,53 @@
 <h1><%= t('.title', resource: resource_name.to_s.humanize) %></h1>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= bootstrap_devise_error_messages! %>
-
-  <div class="form-group">
-    <%= f.label :email %>
-    <%= f.email_field :email, autofocus: true, autocomplete: 'email', class: 'form-control' %>
-  </div>
-
-  <div class="form-group">
-    <%= f.label :password %>
-    <%= f.password_field :password, autocomplete: 'new-password', class: 'form-control' %>
-
-    <small class="form-text text-muted"><%= t('.leave_blank_if_you_don_t_want_to_change_it') %></small>
+<div class="row">
```

Comment:
What does the changes in this file has to do with adding link on navbar?

Reply:
edited PR title